notebooks/ldb_Feature Engineering_kaggleEX.ipynb

```diff
@@ -0,0 +1,2643 @@
+{
```

Comment:
<p>The 2 notebooks are with the name out of pattern.</p><p> </p><p>The raw data are sep by ',' and the decimal is '.', I think you change your csv file.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/neo-empresarial/house_pricing_challenge/pull/42/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='0'/>